services/heartbeat/etcd/store.go

```diff
@@ -302,8 +350,13 @@ func instanceFromKey(key, servicePrefix string) string {
 	)
 }
 
-func servicePrefix(service string) string {
-	return fmt.Sprintf(keyFormat, heartbeatKeyPrefix, service)
+// heartbeats for a service "svc" in env "test" should be stored under
+// "_hb/test/svc"
```

Comment:
nit: just a question to throw out there: if the env is empty, should we do _hb/svc or _hb//svc?